homeworks/konstantin.tsarenko_KTsarenko/homework_7/getFibonacciSequence.test.js

```diff
@@ -0,0 +1,28 @@
+
+const fibonacci = require('./hv_7');
+
+test('first element of sequence to equal 0', () => {
+    expect(fibonacci()[0]).toBe(0);
+});
+
+test('2 element of sequence to equal 1', () => {
+    expect(fibonacci()[1]).toBe(1);
+});
+
+describe('test sequence to be equal fibonacci', () => {
+    const fibonacciArr = fibonacci();
+    for (let i = 2; i < fibonacciArr.length - 3; i++) {
```

Comment:
just didn't get why `i < fibonacciArr.length - 3`? test doesn't need to check tree last element of the result array?

Reply:
this is my fail